examples/load/README.md

```diff
@@ -0,0 +1,155 @@
+# File based task/template/handler definition
+
+This proposal introduces directory based task, template, and handler definition.
+
+## Configuration
+
+```
+[load]
+ enabled = true
+ dir="/path/to/directory"
+```
+
+`dir` specifies the directory where the definition files exist.
+
+The specified directory will check to see if any of three subdirectories `tasks`, `templates`, and `handlers` exists.
+
+The `tasks` directory will contain will contain task tickscripts and and the associated templated task
+definition files (either yaml or json).
+
+The `templates` directory will contain both template tickscripts
+
+The `handlers` directory will contain will contain all topic handler definitions in yaml or json.
+
+## Tasks
+### Tickscript (files with a `.tick` extension)
+
+Tickscripts can be marked up with commented key-value pairs for predefined attributes.
+Those attributes may be any of the following
+
+* `id` - the file name without the tick extension
+* `type` - determined by introspection of the task (stream, batch)
+* `dbrp` - defined using the `dbrp` keyword followed by a specified database and retention policy
+
+For example
+```
+dbrp "telegraf"."autogen"
+
+var measurement string
```

Comment:
This example is actually a template so it can't be placed in the tasks dir.

Reply:
fixed.